integration/sawtooth_integration/tests/test_shutdown_smoke.py

```diff
@@ -214,14 +214,19 @@ def _startup(self):
         try:
             genesis = self._run_genesis()
             self._wait_for_containers_exit_status([genesis])
-            validator_genesis = self._start_validator(genesis, [], [])
+            validator_genesis = self._start_validator(
+                genesis,
+                ['--name', 'validator-0'],
```

Comment:
Hard coded container names will fail on jenkins during parallel builds. It is really annoying since the pass locally always, and then will fail intermittenly on jenkins. My suggestion, until we have more of a dynamic integration testing framework, is that 'validator-0' be a random string.